src/checkout-buttons/strategies/apple-pay/apple-pay-button-strategy.ts

```diff
@@ -216,12 +217,25 @@ export default class ApplePayButtonStrategy implements CheckoutButtonStrategy {
         const { currency: { decimalPlaces } } = state.cart.getCartOrThrow();
         let checkout = state.checkout.getCheckoutOrThrow();
         const availableOptions = checkout.consignments[0].availableShippingOptions;
-        const shippingOptions = availableOptions?.map(option => ({
+        const selectedOption = availableOptions?.filter(option => option.id === checkout.consignments[0].selectedShippingOption?.id);
+        const unSelectedOptions = availableOptions?.filter(option => option.id !== checkout.consignments[0].selectedShippingOption?.id);
```

Comment:
```suggestion
        const unselectedOptions = availableOptions?.filter(option => option.id !== checkout.consignments[0].selectedShippingOption?.id);
```

Reply:
Hi @clopezh, thanks for your suggestion, for sure I will change it.